shimmy/openspiel_wrapper.py

```diff
@@ -0,0 +1,266 @@
+"""Wrapper to convert a openspiel environment into a pettingzoo compatible environment."""
+
+import functools
+from typing import Dict, Optional
+
+import numpy as np
+import pettingzoo as pz
+import pyspiel
+from gymnasium import spaces
+from gymnasium.utils import seeding
+from pettingzoo.utils.env import AgentID
+
+
+class OpenspielWrapper(pz.AECEnv):
```

Comment:
Could we use the new convention of having a version number on wrappers. Add to the DmEnvWrapper as well.
Minor point, while these are wrappers, they are also environment instances. Not sure if this should be called `OpenSpielEnvV0` or `OpenSpielWrapperV0`